app/scripts/models/auth_brokers/base.js

```diff
@@ -146,10 +149,38 @@ define([
      * for email verification. Useful for storing data that may be needed
      * by the verification tab.
      *
+     * @param {object} account
      * @return {promise}
      */
-    persist: function () {
-      return p();
+    persist: function (account) {
+      var self = this;
+
+      return p().then(function () {
+        // verification info is persisted to localStorage so that
+        // the same `context` is used if the user verifies in the same browser.
+        // If the user verifies in a different browser, the
+        // default (direct access) context will be used.
+        var verificationInfo =
+              createSameBrowserVerificationInfo(account, 'context');
```

Comment:
Each SameBrowserVerificationInfo object gets a namespace, I'd like to move the [oauth information that is saved](https://github.com/mozilla/fxa-content-server/blob/9136ab5ee091e33ba76ec185c2a7bee34a6bce4d/app/scripts/models/auth_brokers/oauth.js#L138) to use this object too, I imagine that info would go under the `oauth` namespace.